From 49b12cd0f1a6c904f29c151858b8ca5c4eda1b0f Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Thu, 11 Apr 2019 20:08:57 +0200
Subject: [PATCH] appdata: Keep the app ID same as was in 3.30

Go back to "org.gnome.Boxes.desktop" as in 3.30. The appdata spec is
very clear that the app ID is not supposed to change and should be a
unique identifier for the app across distributions.
---
 data/org.gnome.Boxes.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.Boxes.appdata.xml.in b/data/org.gnome.Boxes.appdata.xml.in
index db8731cd..41e1a2ff 100644
--- a/data/org.gnome.Boxes.appdata.xml.in
+++ b/data/org.gnome.Boxes.appdata.xml.in
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>org.gnome.Boxes</id>
+  <id>org.gnome.Boxes.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>LGPL-2.1+</project_license>
   <name>GNOME Boxes</name>
-- 
2.18.1

